deploy/cluster_role.yaml

```diff
@@ -908,6 +908,7 @@ rules:
   - apps
   resources:
   - deployments
+  - deployments/finalizers
```

Comment:
This also needs to be changed in templates/olm-catalog/kubevirt-hyperconverged/VERSION/kubevirt-hyperconverged-operator.VERSION.clusterserviceversion.yaml.in.

Reply:
Ryan, that looks like a third place already (not counting the original project repository).. have you considered automating this? Even something simple like git precommit-hook or a manually executed script to refresh all the copies...

Reply:
Back when we were building the CSV, we asked components to write the templates as code.  This feature didn't make it into SSP for 2.0, so we carry a copy of the CSV in tree.

The downstream automation is making it's way upstream in https://github.com/kubevirt/hyperconverged-cluster-operator/pull/333 and should replace the current csv generation.

Reply:
It is possible that our generator code got out of date somehow. This is surprising, since it is supposed to use the same YAMLs we used as source. I think we need to check the flow on our side.

Reply:
@rthallisey Yeah, I have no issue in having the copy in tree, but I was wondering why you have three of them..